### PR TITLE
JW-452 : Adding contrail utils

### PR DESCRIPTION
**Pull request by kaweue**

### CODE REVIEW

File: CIScripts/ContrailUtils.ps1 [hunk 1]
```diff
@@ -0,0 +1,92 @@
+function Get-AccessTokenFromKeyStone {
+    Param ([Parameter(Mandatory = $true)] [string] $AuthUrl,
+        [Parameter(Mandatory = $true)] [string] $TenantName,
```
Comment (jablonskim) [inline CIScripts/ContrailUtils.ps1]: Wrong indentation here (in reference to our other PS function)

File: CIScripts/ContrailUtils.ps1 [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    return $Response.'virtual-network'.'uuid'
+}
+
+function Remove-OpenContrailNetwork {
+    Param ([Parameter(Mandatory = $true)] [string] $OpenContrailUrl,
+        [Parameter(Mandatory = $true)] [string] $AuthToken,
+        [Parameter(Mandatory = $true)] [string] $NetworkUuid)
+
+    $RequestUrl = $OpenContrailUrl + "/virtual-network/" + $NetworkUuid
+    Invoke-RestMethod -Uri $RequestUrl -Headers @{"X-Auth-Token" = $AuthToken} -Method Delete
```
Comment (jablonskim) [inline CIScripts/ContrailUtils.ps1]: Please make sure if this function ends with error for example if this network has been deleted already, etc. If no, in my opinion we should check the response

**Review by jablonskim — DISMISSED** [review]

File: CIScripts/ContrailUtils.ps1 (hunk 2 above)
Comment (kaweue) [reply CIScripts/ContrailUtils.ps1]: Exception is thrown if:
- we are not authenticated
- there is no selected network

File: CIScripts/ContrailUtils.ps1 (hunk 1 above)
Comment (kaweue) [reply CIScripts/ContrailUtils.ps1]: Ok I will change that

**Review by jablonskim — APPROVED** [review]

**Pull request merged**